node/README.md

```diff
@@ -189,6 +189,18 @@ things to note:
 
 Both of these cases are handled by the electron-webpack-quick-start example.
 
+### node-gyp and native dependencies
+
+Some node/electron versions are binary incompatible and require rebuilding of
+native node dependencies for electron. In offline mode, it may result in broken ABI.
+If you are seing errors like 
+`The module 'something.node' was compiled against a different Node.js version`,
+then pass `--electron-node-headers` option to flatpak-node-generator and set 
+`npm_config_nodedir` to `flatpak-node/node-gyp/electron-current`.
+
+Also you may want to run `electron-rebuild -f` to make sure that native dependencies
```

Comment:
Nit: where would I be running this? Does it go inside the manifest? After the install but before the build? Might be worth clarifying...

Reply:
Hm. I'd rather remove this note - if native deps aren't rebuilt during `install` or `electron-builder` phase - it's an app flaw not specific to flatpak.

Reply:
Also this is probably not needed for yarn at all - it'll always do rebuild unless `--ignore-scripts` was passed, and during that correct node-header will be picked up thanks to `npm_config_nodedir`, so further rebuilds are redundant.